src/IrisConvert/IrisConvert.php

```diff
@@ -103,23 +121,21 @@ public function createArrayOfSizesToBeConverted($metadata)
     public function convertArrayOfSizes()
     {
         // Remove wp_generate_attachment_metadata action
-        remove_filter('wp_generate_attachment_metadata', [Iris::class, 'convertImageOnUplodad'], 10);
+        // remove_filter('wp_generate_attachment_metadata', [Iris::class, 'convertImageOnUplodad'], 10);
```

Comment:
Should this be disabled still or do we need to ensure it doesn't recall itself within this method?

Reply:
So during my testing I commented this out. I have hunch that when we run the id and batch conversion we do not call the wp_generate_attachment_metadata filter directly, so we need to have it in the process. Unlike the upload which calls that filter on the upload process itself.